PR: https://github.com/odoo/odoo/pull/

From: c7821f528c8655e0fe2326892795fdd59a1ef232
From: Kinjal Mehta
Date: 2018-01-04 12:26:56

Structural Changes: 12
Total Changes: 14

[IMP] sale,sales_team: changed duplicate field label in the same model

================================= pseudo patch: =================================

--- a/addons/sale/models/res_config_settings.py
+++ b/addons/sale/models/res_config_settings.py
@@ -16,19 +16,19 @@ class ResConfigSettings(models.TransientModel):
     group_discount_per_so_line = fields.Boolean("Discounts", implied_group='sale.group_discount_per_so_line')
     module_sale_margin = fields.Boolean("Margins")
     group_sale_layout = fields.Boolean("Sections on Sales Orders", implied_group='sale.group_sale_layout')
-    group_warning_sale = fields.Boolean("Warnings", implied_group='sale.group_warning_sale')
+    group_warning_sale = fields.Boolean("Warnings for Sale", implied_group='sale.group_warning_sale')
     portal_confirmation = fields.Boolean('Online Signature & Payment')
     portal_confirmation_options = fields.Selection([
         ('sign', 'Signature'),
         ('pay', 'Payment')], string="Online Signature & Payment options")
-    module_sale_payment = fields.Boolean("Online Signature & Payment", help='Technical field implied by user choice of online_confirmation')
+    module_sale_payment = fields.Boolean("Sale Payment", help='Technical field implied by user choice of online_confirmation')
     module_website_quote = fields.Boolean("Quotations Templates")
     group_sale_delivery_address = fields.Boolean("Customer Addresses", implied_group='sale.group_delivery_invoice_address')
     multi_sales_price = fields.Boolean("Multiple Sales Prices per Product")
     multi_sales_price_method = fields.Selection([
         ('percentage', 'Multiple prices per product (e.g. customer segments, currencies)'),
         ('formula', 'Prices computed from formulas (discounts, margins, roundings)')
-        ], default='percentage', string="Pricelists")
+        ], default='percentage', string="Pricelists Method")
     sale_pricelist_setting = fields.Selection([
         ('fixed', 'A single sales price per product'),
         ('percentage', 'Multiple prices per product (e.g. customer segments, currencies)'),
@@ -60,7 +60,7 @@ class ResConfigSettings(models.TransientModel):
         domain="[('type', '=', 'service')]",
         oldname='deposit_product_id_setting',
         help='Default product used for payment advances')
-    auto_done_setting = fields.Boolean("Lock Confirmed Orders")
+    auto_done_setting = fields.Boolean("Auto Done Setting")
     module_website_sale_digital = fields.Boolean("Sell digital products - provide downloadable content on your customer portal")
 
     auth_signup_uninvited = fields.Selection([

--- a/addons/sale/models/res_partner.py
+++ b/addons/sale/models/res_partner.py
@@ -10,7 +10,7 @@ class ResPartner(models.Model):
 
     sale_order_count = fields.Integer(compute='_compute_sale_order_count', string='# of Sales Order')
     sale_order_ids = fields.One2many('sale.order', 'partner_id', 'Sales Order')
-    sale_warn = fields.Selection(WARNING_MESSAGE, 'Sales Order', default='no-message', help=WARNING_HELP, required=True)
+    sale_warn = fields.Selection(WARNING_MESSAGE, 'Sales Order Warning', default='no-message', help=WARNING_HELP, required=True)
     sale_warn_msg = fields.Text('Message for Sales Order')
 
     def _compute_sale_order_count(self):

--- a/addons/sale/models/sale.py
+++ b/addons/sale/models/sale.py
@@ -843,7 +843,7 @@ class SaleOrderLine(models.Model):
     price_unit = fields.Float('Unit Price', required=True, digits=dp.get_precision('Product Price'), default=0.0)
 
     price_subtotal = fields.Monetary(compute='_compute_amount', string='Subtotal', readonly=True, store=True)
-    price_tax = fields.Float(compute='_compute_amount', string='Taxes', readonly=True, store=True)
+    price_tax = fields.Float(compute='_compute_amount', string='Total Tax', readonly=True, store=True)
     price_total = fields.Monetary(compute='_compute_amount', string='Total', readonly=True, store=True)
 
     price_reduce = fields.Float(compute='_get_price_reduce', string='Price Reduce', digits=dp.get_precision('Product Price'), readonly=True, store=True)
@@ -855,7 +855,7 @@ class SaleOrderLine(models.Model):
 
     product_id = fields.Many2one('product.product', string='Product', domain=[('sale_ok', '=', True)], change_default=True, ondelete='restrict', required=True)
     product_updatable = fields.Boolean(compute='_compute_product_updatable', string='Can Edit Product', readonly=True, default=True)
-    product_uom_qty = fields.Float(string='Quantity', digits=dp.get_precision('Product Unit of Measure'), required=True, default=1.0)
+    product_uom_qty = fields.Float(string='Ordered Quantity', digits=dp.get_precision('Product Unit of Measure'), required=True, default=1.0)
     product_uom = fields.Many2one('product.uom', string='Unit of Measure', required=True)
     # Non-stored related field to allow portal user to see the image of the product he has ordered
     product_image = fields.Binary('Product Image', related="product_id.image", store=False)
